source/portable/NetworkInterface/STM32Hxx/NetworkInterface.c

```diff
@@ -278,9 +278,6 @@ BaseType_t xNetworkInterfaceInitialise( void )
                 HAL_ETH_DescAssignMemory( &( xEthHandle ), uxIndex, pucBuffer, NULL );
             }
 
-            /* Configure the MDIO Clock */
```

Comment:
Do we need to add ETH_MAC_MDIO_ClkConfig here?

Reply:
No, it calls HAL_ETH_Init in NetworkInterface.c:234, and HAL_ETH_Init calls ETH_MAC_MDIO_ClkConfig in stm32hxx_hal_eth.c:309.